atomic_reactor/inner.py

```diff
@@ -174,6 +174,7 @@ def __init__(self, uri, insecure=False):
         """
         super(DockerRegistry, self).__init__(uri, insecure=insecure)
         self.digests = {}  # maps tags (str) to their digest, if available
+        self.config = None  # stores image config from the registry
```

Comment:
I think you added this to the metadata... really I just wanted it mentioned in the above comment. I don't think we want it in the metadata.